JASP-Engine/JASP/R/correlationbayesian.R

```diff
@@ -1030,13 +1030,14 @@ CorrelationBayesian <- function(jaspResults, dataset=NULL, options, ...) {
     
   
   plotResult <- try(JASPgraphs::PlotRobustnessSequential(
-    dfLines      = dfLines,
-    xName        = bquote(paste(.(gettext("Stretched beta prior width")), ~kappa)),
-    dfPoints     = dfPoints,
-    bfType       = bfPlotType,
-    pointColors  = pointColors,
-    pointFill    = pointFill, 
-    hypothesis   = hypothesisJASPgraphsName
+    dfLines         = dfLines,
+    xName           = bquote(paste(.(gettext("Stretched beta prior width")), ~kappa)),
+    dfPoints        = dfPoints,
+    bfType          = bfPlotType,
+    pointColors     = pointColors,
+    pointFill       = pointFill,
+    hypothesis      = hypothesisJASPgraphsName,
+    plotLineOrPoint = "line"
```

Comment:
I guess here we don't have the correct information to remove `plotLineOrPoint`?

Reply:
No this is a robustness plot and those never use points. Normally, robustness plots always consist of way more than 60 points, for example, the t-tests 400 or 535 points depending on the choice of prior, so the default of 60 works for them. For some reason, this one uses fewer than 60 points.